Refunct/Refunct.asl

```diff
@@ -1,7 +1,7 @@
 state("Refunct-Win32-Shipping")
 {
-    int level  : 0x01C79B64, 0x328, 0x18, 0x2B0, 0x114, 0xA0;
-    int resets : 0x1DCF7DC;
+    int level  : 0x01C79B64, 0xD0, 0x18, 0x2B0, 0xEC, 0xA0;
```

Comment:
This address points to levels for me only after I have gotten the second button, but not a clean process open.  Can you confirm that it works at all time for you?